### PR TITLE
Update coding conventions with route naming

### PR DESCRIPTION
**Pull request by dianafa**
@hakubo @Warkot @rogatty

### CODE REVIEW

File: Ember/README.md [hunk 1]
```diff
@@ -14,6 +14,83 @@
...
+		method: 'POST',
+		path: '/article-preview',
+		handler: articlePreview/
+	}
+```
+Client side:
+```javascript
+	// good
+	this.route('articlePreview', {
+		path: '/article-preview'
```
Comment (hakubo) [inline Ember/README.md]: '/article-preview' is default for articlePreview so you don't have to specify path here

**Comment by rogatty** [comment]
That's not what is in Ember Guides:
https://guides.emberjs.com/v2.3.0/routing/defining-your-routes/#toc_wildcard-globbing-routes
https://guides.emberjs.com/v2.3.0/routing/loading-and-error-substates/

Also, files for routes are dasherized so
```
this.route('article-preview')
```
maps 1:1 to
```
routes/article-preview.js
```

**Comment by RafalWilinski** [comment]
Please note that this will also require some changes in chef-repo for example with image-review.

**Comment by rogatty** [comment]
@RafalWilinski this only changes route names in Ember, not their paths.

**Comment by dianafa** [comment]
@RafalWilinski this changes won't affect the url (paths) only the Ember handler names

**Comment by RafalWilinski** [comment]
Ok, my bad.

**Comment by Warkot** [comment]
Maybe you could also provide a link to aforementioned Ember guidelines in the description. And overall it's a good addition to our repo, good work. :+1:

File: Ember/README.md [hunk 2]
```diff
@@ -14,6 +14,83 @@
...
+		method: 'POST',
+		path: '/articlePreview',
+		handler: article-preview
+	}
+
+	// bad
+	{
+		method: 'POST',
+		path: '/article-preview',
+		handler: articlePreview/
```
Comment (Grunny) [inline Ember/README.md]: While this is also bad for other reasons ;) this trailing slash example should be on the path rather than the handler, right?

Comment (dianafa) [inline Ember/README.md]: thanks!

**Comment by dianafa** [comment]
Can I merge this PR?

**Pull request merged**